fireball.js

```diff
@@ -33,27 +36,34 @@ if (Meteor.isClient) {
       return Session.get("counter");
     },
     mood: function () {
-      var food = Session.get("counter");
+      const food = Session.get("counter");
+      let mood = "The dragon is ";
       if (food === 0) {
         return "Your dragon has run away!";
       }
-      var mood = "The dragon is ";
-      if (food <= 3) {
-        return mood + "angry.";
-      } else if (food <= 5) {
-        return mood + "hungry.";
-      } else if (food === 20) {
-        return mood + "full.";
-      } else {
-        return mood + "content.";
+
+      switch (food) {
```

Comment:
Yep, I think that's fine. Keep in mind that this is a teaching application for beginners so aim towards simplification vs cleverness if that makes sense.